server/app/helper.go

```diff
@@ -0,0 +1,39 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
```

Comment:
This helper file is compiling everything inside the app package when is not compiling the tests, so... we are importing here, in the production binary the mocks a some not-needed code.